rosa_planning/rosa-sts-aws-prereqs.adoc

```diff
@@ -1,10 +1,10 @@
 :_content-type: ASSEMBLY
 include::_attributes/attributes-openshift-dedicated.adoc[]
-:context: rosa-sts-aws-prerequisites
```

Comment:
Do you mean to have both this file and `rosa_getting_started/rosa_getting_started_sts/rosa-sts-aws-prereqs.adoc`? I only see one in the TOC, and they shouldn't have the same contexts like this.